File: Java/src/ClimbingStairsTest.java
import org.junit.Assert;
import org.junit.Test;

public class ClimbingStairsTest {

    @Test
    public void testClimbStairs() {
        ClimbingStairs cs = new ClimbingStairs();

        Assert.assertTrue(cs.climbStairs(1) == 1);

        Assert.assertTrue(cs.climbStairs(3) == 3);

        Assert.assertTrue(cs.climbStairs(5) == 8);
    }

}

File: Kotlin/src/ClimbingStairsTest.kt
import org.junit.Test

class ClimbingStairsTest {

    @Test
    fun testClimbStairs() {

        val cs = ClimbingStairs()

        assert(cs.climbStairs(1) == 1)

        assert(cs.climbStairs(3) == 3)

        assert(cs.climbStairs(5) == 8)
    }

}